layouts/shortcodes/home-section-resources.html

```diff
@@ -0,0 +1,75 @@
+<!-- 
+  Copyright (c) 2019 Eclipse Foundation, Inc.
+
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+
+  Contributors:
+    Yi Liu <yi.liu@eclipse-foundation.org>
+
+  SPDX-License-Identifier: EPL-2.0
+-->
+
+<section class="container">
+  <h2>Resources</h2>
+  <div class="row">
+    <div class="col-md-6 col-sm-12">
+      <div class="flex-column text-center match-height-item background-white padding-20 margin-bottom-30">
+        <div class="flex-grow">
+          <p><img class="img img-responsive margin-auto padding-20" style="height: 150px; width: 150px;" src="images/resources/whitepaper-icon.svg" alt="resource image"></p>
```

Comment:
If alternate text is included on an image, it should be useful text that indicates to the user what it is. Having an alt that says "resources image" for all 4 images provides negative value as it provides confusing context to the images (they aren't all the same)